packages/create-sitecore-jss/src/templates/vue/package.json

```diff
@@ -77,7 +77,7 @@
     "babel-eslint": "~10.0.3",
     "chokidar": "~3.1.1",
     "cross-env": "~6.0.0",
-    "eslint": "^7.4.0",
+    "eslint": "^6.8.0",
```

Comment:
Interesting. So the solution is to downgrade eslint? We should probably try to remain on 7.x to remain in sync with rest of the repo.

Reply:
Exactly, it's a solution, because vue-eslint-plugin requires to use version <7

Reply:
Good point, it's possible try to upgrade plugin, I will do it

Reply:
Oh, no. Unfortunately, it's impossible, I checked it before, we have latest version of `@vue/cli-plugin-eslint` and they will support eslint 7 only in the next version, when it will be released

Reply:
Got it. It's fine then. Might be good to add this to the backlog to upgrade these when released.